packages/cli/src/main_parser_factory.ts

```diff
@@ -8,7 +10,12 @@ import { createConfigureCommand } from './commands/configure/configure_command_f
  * Creates main parser.
  */
 export const createMainParser = (): Argv => {
+  const absolutePath = './node_modules/@aws-amplify/backend-cli';
```

Comment:
we should use `import.meta.url` to determine this.
But I suggest to wait until @Amplifiyer 's PR goes in that adds components for that to platform core.

Reply:
You can now use 

https://github.com/aws-amplify/amplify-backend/blob/70685f36bdd089711fab3ba6eab0c375dad35827/packages/platform-core/src/package_json_reader.ts#L8

See example 

https://github.com/aws-amplify/amplify-backend/blob/70685f36bdd089711fab3ba6eab0c375dad35827/packages/cli/src/commands/sandbox/sandbox_command_factory.ts#L57-L59

Reply:
Done! 4c50d21d4